venia-integration-tests/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "pwa-automation-tests",
```

Comment:
Consistency - some places it's `pwa-automation-tests`, `venia-e2e-tests`, or `venia-integration-tests`.